cns/NetworkContainerContract.go

```diff
@@ -69,6 +69,12 @@ const (
 	Vxlan = "Vxlan"
 )
 
+// Address Types
+const (
```

Comment:
what's difference between `addressType` and `isDefaultInterface`? Why require both?

Reply:
address type makes sense since we're not getting the interface info for default, if secondary we'll get the mac address

Reply:
adressType doesn't make sense to me..tommorow cni has to support IB nic and addressType will be IB for that? nicType makes more sense

Reply:
ok gonna use NICType then

Reply:
agree with Tamilmani, isDefaultInterface and NicType as Default is confusing. So in Swift 2.0 my NicType = default will be eth0, but my eth1 will isDefaultInterface

Reply:
can be `defaultGatewayNic` instead of `defaultNic`

Reply:
also I think you should make this an enum as well
```
type NICType string

// NIC Types
const (
	NICTypeDefault   NICType = "Default"
	NICTypeSecondary NICType = "Secondary"
)
```

Reply:
renamed to Infra and Secondary NICTypes